dlatk/dlaWorker.py

```diff
@@ -80,9 +80,18 @@ def __init__(self, db_type, corpdb, corptable, correl_field, mysql_config_file,
     
     def prepare_corpdb(self):
 
-        if (".csv" in self.corptable) or (self.db_type == "sqlite"):
+        if (self.corptable and ".csv" in self.corptable) or (self.db_type == "sqlite"):
```

Comment:
Do we need this check? Because `DLAWorker.__init__()` assigns `corptable` and there's a default corptable in `dlatkInterface.py`.

Reply:
Honestly, I can't remember why I put this in there. If `self.corptable` is `None` then the code will fail at lines 92 or 99.